pkg/apis/servicecatalog/v1alpha1/register.go

```diff
@@ -62,6 +62,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	)
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
 	scheme.AddKnownTypes(schema.GroupVersion{Version: "v1"}, &metav1.Status{})
-
+	scheme.AddFieldLabelConversionFunc("servicecatalog.k8s.io/v1alpha1", "ServiceClass", ServiceClassFieldLabelConversionFunc)
```

Comment:
I did not realize these functions existed - it makes sense that they would, though.